vm/qemu/qemu.go

```diff
@@ -608,11 +614,20 @@ func (inst *instance) Run(timeout time.Duration, stop <-chan bool, command strin
 }
 
 func (inst *instance) Diagnose() ([]byte, bool) {
-	select {
-	case inst.diagnose <- true:
-	default:
+	msg := fmt.Sprintf("%s Registers:\n", time.Now().Format("15:04:05 "))
+	ret := []byte{}
+	ret = append(ret, []byte(msg)...)
+	for cpu := 0; cpu < inst.cfg.CPU; cpu++ {
+		regs, err := inst.hmp("info registers", cpu)
+		if err == nil {
+			log.Logf(0, "VM-%v\n%v", inst.index, regs)
```

Comment:
This dumps registers of all CPUs, not just the one with a crash, and sets of registers are different - the kernel does not dump FPU on PPC64, also this dumps QEMU's version of the crash which might differ from the VM's point of view. Still remove? Make optional (then, how)?

Reply:
We append it to the crash log on the following line. Why do we need to duplicate this in the syz-manager log?
This will duplicate register values for each each CPU for each crash in manager log on the console. I don't see how this is useful. In lots of use cases nobody is even reading manager log.

Reply:
ah the manager log is the objection, ok.